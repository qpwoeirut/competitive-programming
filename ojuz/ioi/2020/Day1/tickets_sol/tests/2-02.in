df804a0e-e4b1-4b8d-a0f5-83401a72b5a3
6 6 1
100644700 183525295 271589520 388623007 858346721 984904151
52152924 65524343 77102223 233942597 331800721 961494861
54466099 136845379 483732779 838382640 935832309 983824097
100853681 287600049 325835150 375487898 668222749 728211762
27896281 111275853 397178855 412901485 625073635 687196058
73592061 183265584 269601446 348790940 490117669 728745498
