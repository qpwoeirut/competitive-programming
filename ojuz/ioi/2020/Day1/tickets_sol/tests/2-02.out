131b432f-1f84-4f8f-9d55-41e868a6bc5d
OK
2727881086
-1 -1 -1 -1 -1 0
-1 -1 -1 -1 -1 0
-1 -1 -1 -1 -1 0
0 -1 -1 -1 -1 -1
0 -1 -1 -1 -1 -1
0 -1 -1 -1 -1 -1
